packages/engine/src/mocap/MotionCaptureSystem.ts

```diff
@@ -168,7 +168,7 @@ const execute = () => {
               MotionCaptureRigComponent.rig[boneName].w[entity]
             )
             .normalize(),
-          getState(ECSState).deltaSeconds * 50
+          getState(ECSState).deltaSeconds * 35
```

Comment:
what is this multiplier doing exactly?

Reply:
this increases the alpha of the lerp function, the changes here are just me playing around with different different rates of change